src/pages/infopages/SignedOutPage.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Button, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { Link, Redirect, useLocation } from 'react-router-dom';
+import { RootState } from '../../redux/store';
+import { LocalStorageKey } from '../../utils/constants';
+import { dataTestId } from '../../utils/dataTestIds';
+import { useAuthentication } from '../../utils/hooks/useAuthentication';
+import { UrlPathTemplate } from '../../utils/urlPaths';
+
+const SignedOutPage = () => {
+  const { t } = useTranslation();
+  const { handleLogin } = useAuthentication();
+  const location = useLocation();
+  const user = useSelector((store: RootState) => store.user);
+
+  if (user) {
+    return <Redirect to={UrlPathTemplate.Home} />;
+  }
+
+  return (
+    <Box sx={{ m: '4rem 1rem 1rem 1rem' }}>
```

Comment:
Kan vel ha shorthand top | left&right | bottom her. 
Kunne vel egentlig også hatt '4rem 1rem', med mindre det er en grunn til at bottom bare skal ha 1rem? Synes selv det ser litt bedre ut med lik spacing over og under på så liten skjerm hvor dette har noe å si.

```suggestion
    <Box sx={{ m: '4rem 1rem 1rem' }}>
```

Reply:
<img width="471" alt="image" src="https://github.com/BIBSYSDEV/NVA-Frontend/assets/10809754/5e47b7e5-fb5a-4e1d-80f3-2b4730791e51">

vs 

<img width="467" alt="image" src="https://github.com/BIBSYSDEV/NVA-Frontend/assets/10809754/654101d2-cfd0-4b94-8ced-e7fffd2a1ef9">

Reply:
Enig i at `'4rem 1rem'` er bedre!